frontend/src/components/Layouts/HomePageComponent/components/CategoryOutput/CategoryOutput.module.scss

```diff
@@ -28,12 +29,25 @@
   align-items: flex-end;
 }
 
-@include md {
+@include lg {
+  .discounted {
+    display: grid;
+    grid-template-columns: repeat(8, 24%);
+    scroll-snap-type: x mandatory;
+    -webkit-overflow-scrolling: touch;
+    max-width: 100vw;
+  }
   .cardOutput {
     overflow-x: auto;
     gap: 14px;
   }
 }
+@media screen and (max-width: 744px) {
```

Comment:
Can we use existing breakpoints? Or we need another one?

Reply:
For this purpose no, we can't

Reply:
Then it would be better to add this breakpoint to the variables and make a new mixin